### DISCUSSION TITLE
Experiences with very small Deployments?

### DISCUSSION THREAD

**Discussion by kvndrsslr** [coder/coder#10180] [General] [answered]
Hey, I am currently looking at Coder for a very small team (expecting at most 3-4 concurrent workspaces at a time, about 25 users total). All the information about scaling and installing coder is generally about scaling upwards of 1000s of users, not down.

So I figured to ask here, what are your experiences with hosting a small coder deployment on ec2? Ideally I want to host `code-server`, `coder v2 OSS` and Docker workspaces on the same (small) EC2 instance to cut down on costs.

FYI I have evaluated other options, but since we are using self-hosted Gitlab, none of the other major players in this space are viable for us.

**Comment by matifali** [comment, marked as answer]
Do you plan to use the same machine to host Coder and workspaces? I think Coder will not need more than 1GB of RAM, but for your workspaces, it all depends on what your devs do with them. See [this](https://coder.com/blog/how-our-development-team-shares-one-giant-bare-metal-machine) related blog post for how we dogfood Coder.

**Reply by kvndrsslr** [reply]
Thanks, that helped!
